src/landingPage/ModuleIntro.tsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 
-export default function ModuleIntro(props) {
+export default function ModuleIntro(props:
```

Comment:
It would be helpful to add some documentation here for the props. I found [this file](https://github.com/sandboxnu/a-eye/blob/master/src/aboutPage/ProfileCard.tsx) that has an example and [this looks to be a decent tutorial for adding JSDoc to react components](https://www.inkoop.io/blog/a-guide-to-js-docs-for-react-js/). Whether this is necessary is ultimately up to @krisharma though!

Reply:
I'd go through and do this for the other components you've defined if we want to add the documentation

Reply:
Yes JSDoc would be great! Just good practice for us moving forward - thanks for catching this Jake